src/pkg/packager/create.go

```diff
@@ -382,15 +397,21 @@ func (p *Packager) addComponent(component types.ZarfComponent) (*types.Component
 		spinner := message.NewProgressSpinner("Loading %d K8s manifests", manifestCount)
 		defer spinner.Success()
 
-		if err := utils.CreateDirectory(componentPath.Manifests, 0700); err != nil {
-			return nil, fmt.Errorf("unable to create manifest directory %s: %w", componentPath.Manifests, err)
-		}
-
 		// Iterate over all manifests.
 		for _, manifest := range component.Manifests {
 			for idx, f := range manifest.Files {
 				// Copy manifests without any processing.
 				spinner.Updatef("Copying manifest %s", f)
+				if utils.IsURL(f) {
+					parsed, err := url.Parse(f)
+					if err != nil {
+						return nil, fmt.Errorf("unable to parse manifest URL %s: %w", f, err)
+					}
+					name := filepath.Base(parsed.Path)
```

Comment:
This won't necessarily be the filename... it could be nothing or a directory tree.